benefits/oauth/apps.py

```diff
@@ -0,0 +1,18 @@
+"""
+The oauth application: Implements OAuth-based authentication
+"""
+from django.apps import AppConfig
+
+
+class OAuthAppConfig(AppConfig):
+    name = "benefits.oauth"
+    label = "oauth"
+    verbose_name = "Benefits OAuth"
+
+    def ready(self):
+        from .client import oauth, register_providers
```

Comment:
You meant to do this import in a function? If so, mind adding a comment explaining why?